src/librbd/exclusive_lock/PreReleaseRequest.cc

```diff
@@ -46,6 +46,7 @@ PreReleaseRequest<I>::~PreReleaseRequest() {
   if (!m_shutting_down) {
     m_image_ctx.state->handle_prepare_lock_complete();
   }
+  delete m_on_finish;
```

Comment:
There are several options here.

1: Change the code to appease coverity
2: Mark the issue as a false positive in the coverity interface
3: Add a coverity suppression statement to silence coverity
4: Ignore the error, since we know it is a false positive.